### PR TITLE
Update SentinelOne-V2.yml

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/CRTXDOC-3017

## Description
Initial review of SentinelOne pack.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
Comments/edits on yml file:
- There is a command - sentinelone-get-white-list
    The description is: Lists all exclusion items that match the specified input filter.
    Shouldn't this command list all the included items? Not the excluded items?
    Can you please check with the partner to see if this description is correct? (@kgal-pan is checking on this)
    
 - Line 104 - I don't see that there is an API Token (recommended) parameter.
     additionalinfo: Use the "API Token (Recommended)" parameter instead.

- Line 386
  The example does not match the predefined list.

- Line 1400, 1402, 1406, 1862
   Should the description be: A comma-separated list of....

**Comment by kgal-pan** [comment]
@munna-metron @saurabh-metron - Can you please review and approve these changes?

**Comment by ShirleyDenkberg** [comment]
In the pack readme, the last line is:
Please contact the partner directly via the support link on the right.
But there is no link.  Either provide a link if it is needed or remove this line (and perhaps the line before it).

**Comment by munna-metron** [comment]
Hi @ShirleyDenkberg ,
These are my comments on the PR:
- **SentinelOne-get-white-list**: This command was not added by us and is a bit older. When I cross-checked the endpoint details in the API documentation provided by SentinelOne, they were correct. I suggest changing the name of this command to something like SentinelOne-get-exclusions-list
-- Attached a screenshot of API doc for the same.
- **Additional Info**: Use the "API Token (Recommended)" parameter instead. Feel free to remove the "(Recommended)" string from the additional info.
- **Classification Query Parameter in sentinelone-get-threats**: Correct, the example and the predefined options are not matching. Cross-checked the classification query parameter in the official SentinelOne API docs. The classification field is an array and this needs to be a text box that will accept comma-separated values instead of a dropdown.
-- Attached screenshot for the same.
- **Support Link in README.md**: You can find the support link in the `pack-metadata.json` file. Additionally, update the README with this line in #L16: "Please contact the partner directly via the support [link](https://www.sentinelone.com/support/)."
- **README Update**: Also please update the SentinelOne-V2/README.md file according to the changes made in the YAML files.
Thanks!
image (55)
image (54)

File: Packs/SentinelOne/README.md [hunk 1]
```diff
@@ -7,10 +7,13 @@ This pack enables you to
...
 The **Sentinel One - Endpoint data collection** playbook collects endpoint information by using SentinelOne commands.
+
 The pack includes the **SentinelOne v2** integration and the **Sentinel One - Endpoint data collection** playbook.
+
 ## How does this pack work?
 Create an instance of the **SentinelOne v2** integration and start fetching information from the SentinelOne database.
 
-Note: Support for this Pack moved to the partner on March, 23, 2023.
+Note: Support for this pack moved to the partner on March, 23, 2023.
 Please contact the partner directly via the support link on the right.
```
Comment (ShirleyDenkberg) [inline Packs/SentinelOne/README.md:19]:
```suggestion
Please contact the partner directly via the support [link](https://www.sentinelone.com/support/).
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 @kgal-pan @munna-metron 
Not sure what to do with this comment:
I asked: Line 386 - The example does not match the predefined list.
Partner said: Correct, the example and the predefined options are not matching. Cross-checked the classification query parameter in the official SentinelOne API docs. The classification field is an array and this needs to be a text box that will accept comma-separated values instead of a dropdown.
-- Attached screenshot for the same. (see partner's comment above).

I can change the description to: List of threat classifications to search.
But should the predefined values be removed? or Should the example be removed?

And I did not get a response to:
Line 1400, 1402, 1406, 1862
Should the description be: A comma-separated list of....

**Comment by munna-metron** [comment]
Hi @ShirleyDenkberg ,

1. Since this one of first commands built for this integration, a lot has changed from then, not sure of this API used to contain some predefined values earlier but now it supports comma-separated values, which can be of any combination (the list keeps evolving, it shouldn't be static, imo), the recommendation is to remove the predefined options and make the classification a text field where the user can provide comma-separated values, which we will then parse into a list. This should do the trick, and there is no need to update the examples. I hope this makes sense.
2. About lines 1400, 1402, 1406, and 1862, the current descriptions will suffice.
Thanks

**Comment by ShirleyDenkberg** [comment]
@munna-metron Thanks for your response.  I removed the predefined values.

**Pull request closed without merging**